src/vaadin-grid-pro-edit-column.html

```diff
@@ -53,12 +53,14 @@
            *   - `rowData.expanded` Sublevel toggle state.
            *   - `rowData.level` Level of the tree represented with a horizontal offset of the toggle button.
            *   - `rowData.selected` Selected state.
+           * @type {!GridBodyRenderer | undefined}
            */
           editModeRenderer: Function,
```

Comment:
Maybe include `| null` too to be consistent with vaadin-grid which has `@type {GridBodyRenderer | null | undefined}`?
https://github.com/vaadin/vaadin-grid/blob/master/src/vaadin-grid-column.html#L662

Reply:
Done.